semgrep-core/tests/java/partial_try.java

```diff
@@ -0,0 +1,36 @@
+package org.apache.cassandra.index;
```

Comment:
This is the `.java` file for `partial_try.sgrep`, but I don't see a corresponding `.java` file for `partial_finally.sgrep`. Is it supposed to be the target for both?